Documentation/Logo.md

```diff
@@ -1,56 +1,79 @@
-# Startup Logos
+# Startup Logo / Animation
 
-This firmware supports a user created bootup logo.
-By default, there is _not_ one included in the firmware. This means that once flashed they generally stay. If you want no logo again, you would have to flash a blank image to the bootup logo. 
+When the device starts, you can have it optionally show either a static image or an animation. You can also set if these should stay on the screen or dismiss after some amount of time.
+These can be an elegant way to personalise your device or just mark it as your one at a meetup where there may be multiple.
+
+All devices supported by IronOS support this logo, and follow a similar process for setting one up. Please read the below general information as well as any model specific notes.
+
+Bootup logos are stored at the end of the flash storage in the Iron; next to the user settings. By locating them at the end of storage they are not erased during the normal firmware upgrade process. Once a logo is set it should stay (unless we need to change things in the main firmware); so to erase your logo you will also find that we generate an erase file. Alternatively your method of flashing _may_ support doing a full erase flash which will also work for this.
 
 ## Generating the Logo files
 
-There are community logo's already converted and ready to use in [IronOS-Meta/releases](https://github.com/Ralim/IronOS-Meta/releases).
+Because logo's are stored at a fixed location in the device's internal flash; we can use the same method to flash these as you would normal firmware.
```

Comment:
```suggestion
Because logos are stored at a fixed location in the device's internal flash; we can use the same method to flash these as you would normal firmware.
```